core/src/apps/homescreen/homescreen.py

```diff
@@ -53,7 +53,14 @@ def do_render(self) -> None:
 
         # homescreen with shifted avatar and text on bottom
         ui.display.avatar(48, 48 - 10, self.get_image(), ui.WHITE, ui.BLACK)
-        ui.display.text_center(ui.WIDTH // 2, 220, self.label, ui.BOLD, ui.FG, ui.BG)
+        if utils.MODEL == "1" or utils.MODEL == "R":
```

Comment:
This is Python, so we can use the following:

``` python
if utils.MODEL in ["1", "R"]:
```

Reply:
s/`[]`/`()`, tuples are preferred for static lists of options
or `in "1R"` works for single-letter models too :)